config/jobs/containerd/containerd/containerd-postsubmit-jobs.yaml

```diff
@@ -0,0 +1,130 @@
+            resources:
+              limits:
+                cpu: 4
+                memory: 6Gi
+              requests:
+                cpu: 4
+                memory: 6Gi
+      annotations:
+        testgrid-dashboards: sig-node-containerd,containerd-postsubmits
+        testgrid-tab-name: post-containerd-build
+        description: "builds development in progress branch of upstream containerd"
+
+    - name: post-containerd-build-1-6
+      labels:
+        preset-service-account: "true"
+      cluster: k8s-infra-prow-build
+      decorate: true
+      branches:
+        - release/1.6
+      spec:
+        containers:
+          - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230616-e730b60769-master
+            command:
+              - runner.sh
+            args:
+              - test/build.sh
+            env:
+              - name: DEPLOY_DIR
+                value: release-1.6
+              - name: DEPLOY_BUCKET
+                value: k8s-staging-cri-tools
+            resources:
+              limits:
+                cpu: 4
+                memory: 6Gi
+              requests:
+                cpu: 4
+                memory: 6Gi
+      annotations:
+        testgrid-dashboards: sig-node-containerd,containerd-postsubmits
+        testgrid-tab-name: post-containerd-build-1.6
+        description: "builds release/1.6 branch of upstream containerd"
+
+    - name: post-containerd-build-1-7
+      labels:
+        preset-service-account: "true"
+      cluster: k8s-infra-prow-build
+      decorate: true
+      branches:
+        - release/1.7
+      spec:
+        containers:
+          - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230616-e730b60769-master
+            command:
+              - runner.sh
+            args:
+              - test/build.sh
+            env:
+              - name: DEPLOY_DIR
+                value: release-1.7
+              - name: DEPLOY_BUCKET
+                value: k8s-staging-cri-tools
+            resources:
+              limits:
+                cpu: 4
+                memory: 6Gi
+              requests:
+                cpu: 4
+                memory: 6Gi
+      annotations:
+        testgrid-dashboards: sig-node-containerd,containerd-postsubmits
+        testgrid-tab-name: post-containerd-build-1.7
+        description: "builds release/1.7 branch of upstream containerd"
+
+    - name: post-containerd-build-test-images
+      labels:
+        preset-service-account: "true"
+        preset-dind-enabled: "true"
+      cluster: k8s-infra-prow-build
+      decorate: true
+      branches:
+        - main
+      spec:
+        containers:
+          - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230616-e730b60769-master
+            command:
+              - runner.sh
+            args:
+              - test/build-test-images.sh
+            env:
+              - name: DEPLOY_DIR
+                value: main
+              - name: PROJECT
```

Comment:
```suggestion
              - name: DEPLOY_BUCKET
```

Reply:
This script pushes images and not blobs. Have a look at https://github.com/containerd/containerd/blob/main/test/build-test-images.sh#L25 and the last 2 lines

Reply:
Then, shouldnt it be PROJECT in the periodics also? https://github.com/kubernetes/test-infra/pull/30025/files#diff-669927a638396eb51d888d0752bf589ea0ee622ac29fab684feda313eb88edd7R129

Reply:
@upodroid Can you check this?, Shouldnt the postsubmit and periodics be the same.